lib/samples/show_portal_user_info/show_portal_user_info_sample.dart

```diff
@@ -0,0 +1,168 @@
+                return const Text('Authenticating...');
+              }
+
+              // If the portal load failed with an error, display the error.
+              if (snapshot.hasError) {
+                return Text('Error: ${snapshot.error}');
+              }
+
+              // If the portal load succeeded, display the portal information.
+              final titleStyle = Theme.of(context).textTheme.titleMedium;
+              return SingleChildScrollView(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      '${_portal.user?.fullName} Profile',
+                      style: Theme.of(context).textTheme.titleLarge,
+                    ),
+                    const SizedBox(height: 20.0),
+                    _userThumbnail != null
+                        ? Image.memory(_userThumbnail!)
+                        : const Icon(Icons.person),
+                    Text('Full name', style: titleStyle),
+                    Text(_portal.user?.fullName ?? ''),
+                    Text('Username', style: titleStyle),
+                    Text(_portal.user?.username ?? ''),
+                    Text('Email', style: titleStyle),
+                    Text(_portal.user?.email ?? ''),
+                    Text('Description', style: titleStyle),
+                    Text(_portal.user?.userDescription ?? ''),
+                    Text('Access', style: titleStyle),
+                    Text(_portal.user?.access.name ?? ''),
+                    const Divider(),
+                    _organizationThumbnail != null
+                        ? Image.memory(_organizationThumbnail!)
+                        : const Icon(Icons.domain),
+                    Text('Organization', style: titleStyle),
+                    Text(_portal.portalInfo?.organizationName ?? ''),
+                    Text('Can find external content', style: titleStyle),
+                    Text('${_portal.portalInfo?.canSearchPublic}'),
+                    Text('Can share items externally', style: titleStyle),
+                    Text('${_portal.portalInfo?.canSharePublic}'),
+                    Text('Description', style: titleStyle),
+                    Text(_portal.portalInfo?.organizationDescription ?? ''),
+                  ],
+                ),
+              );
+            }),
+      ),
+    );
+  }
+
+  // Load the user and organization thumbnails.
+  void loadThumbnails() {
+    _portal.user?.thumbnail?.loadBytes().then((bytes) {
+      setState(() => _userThumbnail = bytes);
+    });
+    _portal.portalInfo?.thumbnail?.loadBytes().then((bytes) {
+      setState(() => _organizationThumbnail = bytes);
+    });
+  }
+}
+
+extension on LoadableImage {
+  // A helper method to load the image bytes.
+  Future<Uint8List> loadBytes() async {
+    await load();
+    return image!.getEncodedBuffer();
+  }
+}
```

Comment:
Is it qorth a discussion if we provide this in the API if it's going to be required in most use cases to save the extension being needed per application?

Reply:
Nice, it'll clean it up like we do for the basemap and tile cache thumbnails 👍